### ISSUE TITLE
Styling the add comments section below article

### ISSUE THREAD

**Issue by KajalDeore04** [OkenHaha/react-blog#4]
I want to style this part as no one can actually see the text block and it is not aligned properly
![Screenshot 2024-12-25 102803](https://github.com/user-attachments/assets/7b85fbd8-d7a4-4b79-a2ad-d6f04795621d)

Please assign me this task under SWOC
**I will be able to contribute after 1st of Jan as it counts then**

**Comment by YogyashriPatil** [comment]
Hi team,

I've added my work on the "Styling the add comments section below article" feature. I have developed a markdown template to beautify the comment section, which will make it more user-friendly and visually appealing. The name, comment field, and a submit button are included in the markdown template and also completed the backend.

I'm waiting for your feedback and any improvements.

Thanks!
Best, Yogyashri

**Comment by OkenHaha** [comment]
@KajalDeore04 and @YogyashriPatil I'm assigning this issue to both of you so please look into it. Thank you for your contribution.

**Comment by KajalDeore04** [comment]
@OkenHaha I will start working ASAP after 1st of Jan

**Comment by OkenHaha** [comment]
@KajalDeore04 Take your time and please provide a screenshot of your work after your completion

**Comment by KajalDeore04** [comment]
@OkenHaha please assign the level label to this

**Comment by OkenHaha** [comment]
@KajalDeore04 the `level` tags are used for issues related with the backend and code architecture and structure. Like API calling and fetching is considered `level 1` since the issue you want to work on is UI there's no need for `level` label to be assigned. I hope you understand.

**Comment by KajalDeore04** [comment]
@OkenHaha Yeah I understand but other project admins have actually assigned those labels irrespective of what issue they are solving so...

**Comment by OkenHaha** [comment]
@KajalDeore04 Alright, I understand, it just a fancy stuff to let it know what kind of issue is being worked on, have you check what the label said for the other project admins? I'd like to know, in the mean time, I'll find a new label to add for UI. Please go on ahead with your work.

**Comment by KajalDeore04** [comment]
@OkenHaha It's either beginner, intermediate or advance. Don't wanna sound rude or anything but I think those labels ensure we get points for our work and that was the sole reason I asked for it.

**Comment by OkenHaha** [comment]
@KajalDeore04 Let me check again, in the mean time I'll assign it as Level 1
